src/invoke-lambda/index.js

```diff
@@ -46,8 +46,16 @@ module.exports = function invokeLambda (params, callback) {
         ? `${join(src, 'vendor')}:${process.env.PYTHONPATH}`
         : join(src, 'vendor')
 
+      let srcParts = src.split('/')
+      let functionName = srcParts.pop()
+      let functionType = srcParts.pop().toUpperCase()
+
       let defaults = {
-        __ARC_CONTEXT__: JSON.stringify({}), // TODO add more stuff to sandbox context
+        __ARC_CONTEXT__: JSON.stringify({
+          'functionName': `sandbox-${functionName}${functionType}`,
+          'functionVersion': '$LATEST',
+          // todo awsRequestId // needs to be a first class thing right now it only exists in request context for websockets
+        }),
```

Comment:
A potential issue here: how does the context object change between `HTTP`, `HTTP` in `REST` mode (aka `httpv1` in Arc parlance), and `REST` API Gateways? I'm almost certain they've got diff stuff in there.